_data/risc.yml

```diff
@@ -0,0 +1,77 @@
+#
```

Comment:
Since the format we use for incoming and outgoing events is slightly different, any thoughts on breaking those into separate files? e.g. `risc_outgoing.yml` and `risc_incoming.yml`. Corrects for the issue where someone adding an outgoing event copies the first one in the file and pastes it at the bottom and winds up missing a bunch of keys.

Reply:
Makes sense! Will give it a shot. Still combine them in the resulting `/data/risc.json`?

Reply:
I like that idea.

Reply:
PTAL @ ba759b78fd0d7c3c28c473984a39c91ff26e357f